docs/developers/keystore.md

```diff
@@ -0,0 +1,56 @@
+---
+id: keystore
+title: MultiversX Smart Contracts
```

Comment:
Title is not all right. Perhaps `MultiversX Wallet: keystore files`?

Reply:
yes, typo